test/integration/shoots/creation/create_test.go

```diff
@@ -0,0 +1,228 @@
+		validateFlags()
+		testLogger = logger.AddWriter(logger.NewLogger(*logLevel), GinkgoWriter)
+
+		shoot := prepareShoot()
+		shootGardenerTest, err = NewShootGardenerTest(*gardenerKubeconfigPath, shoot, testLogger)
+		Expect(err).To(BeNil())
+		err := shootGardenerTest.SetupShootWorker(workerZone)
+		Expect(err).To(BeNil())
+		Expect(len(shootGardenerTest.Shoot.Spec.Provider.Workers)).Should(BeNumerically("==", 1))
+
+		// override default worker settings with flags
+		if shootMachineType != nil && len(*shootMachineType) > 0 {
+			shootGardenerTest.Shoot.Spec.Provider.Workers[0].Machine.Type = *shootMachineType
+		}
+
+		if shootMachineImageName != nil && len(*shootMachineImageName) > 0 {
+			shootGardenerTest.Shoot.Spec.Provider.Workers[0].Machine.Image.Name = *shootMachineImageName
+		}
+
+		if shootMachineImageVersion != nil && len(*shootMachineImageVersion) > 0 {
+			shootGardenerTest.Shoot.Spec.Provider.Workers[0].Machine.Image.Version = *shootMachineImageVersion
+		}
+
+		gardenerTestOperation, err = NewGardenTestOperation(shootGardenerTest.GardenClient, testLogger)
+		Expect(err).To(BeNil())
+	}, InitializationTimeout)
+
+	CIt("Create & Reconcile Shoot", func(ctx context.Context) {
+		testLogger.Infof("Creating shoot %s in namespace %s", shootGardenerTest.Shoot.Name, *projectNamespace)
+		if err := printShoot(shootGardenerTest.Shoot); err != nil {
+			testLogger.Fatalf("Cannot decode shoot %s: %s", shootGardenerTest.Shoot.Name, err)
+		}
+
+		shootObject, err := shootGardenerTest.CreateShoot(ctx)
+		if err != nil {
+			if shootObject != nil {
+				if err := gardenerTestOperation.AddShoot(ctx, shootObject); err != nil {
+					testLogger.Errorf("Cannot add shoot %s to test operation: %s", shootGardenerTest.Shoot.Name, err.Error())
+				}
+				gardenerTestOperation.DumpState(ctx)
+			}
+			testLogger.Fatalf("Cannot create shoot %s: %s", shootGardenerTest.Shoot.Name, err.Error())
+		}
+		testLogger.Infof("Successfully created shoot %s", shootGardenerTest.Shoot.Name)
+
+		if err := gardenerTestOperation.AddShoot(ctx, shootObject); err != nil {
+			testLogger.Fatalf("Cannot add shoot %s to test operation: %s", shootGardenerTest.Shoot.Name, err.Error())
+		}
+
+		err = gardenerTestOperation.DownloadKubeconfig(ctx, gardenerTestOperation.SeedClient, gardenerTestOperation.ShootSeedNamespace(), "gardener", *shootKubeconfigPath)
+		if err != nil {
+			testLogger.Fatalf("Cannot download shoot gardenerKubeconfigPath: %s", err.Error())
+		}
+		err = gardenerTestOperation.DownloadKubeconfig(ctx, gardenerTestOperation.GardenClient, gardenerTestOperation.Seed.Spec.SecretRef.Namespace, gardenerTestOperation.Seed.Spec.SecretRef.Name, *seedKubeconfigPath)
+		if err != nil {
+			testLogger.Fatalf("Cannot download seed gardenerKubeconfigPath: %s", err.Error())
+		}
+		testLogger.Infof("Finished creating shoot %s", shootGardenerTest.Shoot.Name)
+	}, CreateAndReconcileTimeout)
+})
+
+func prepareShoot() *gardencorev1alpha1.Shoot {
```

Comment:
we need to enable the nginx addon as it is necessary for the shootapp test `addons.nginx-ingress.enabled: false`

Reply:
ok, makes sense, as you are using the same shoot for the apptest in the test machinery